supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -242,6 +271,15 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
     asyncIo.writeAsync(cloudPath, gcsTransferLibrary, Seq(CloudStorageOptions.withMimeType("text/plain")))
   }
 
+  override def uploadGcsEgressCheckScript(createPipelineParameters: CreatePipelineParameters,
+                                          cloudPath: Path,
+                                          transferLibraryContainerPath: Path,
+                                          gcsTransferConfiguration: GcsTransferConfiguration,
+                                          referenceInputsToMountedPathsOpt: Option[Map[PipelinesApiInput, String]]): Future[Unit] = {
+    val content = generateGcsEgressCheckScript(createPipelineParameters.inputOutputParameters.fileInputParameters)(gcsTransferConfiguration)
+    asyncIo.writeAsync(cloudPath, s"source '$transferLibraryContainerPath'\n\n" + content, Seq(CloudStorageOptions.withMimeType("text/plain")))
```

Comment:
if this isn't using anything in the transfer library it could forego the `source '$transferLibraryContainerPath'` bit